bayesian/utility/security_vendor.py

```diff
@@ -164,13 +247,26 @@ def get_message(self):
         :return: Message string.
         """
         logger.info("Generating Message String.")
-        message = f"{self.package} - {self.version} has {self.public_vul} " \
-                  f"known security vulnerability"
+        message = f"{self.package} - {self.version} has "
+
+        if self.public_vul and self.pvt_vul:
+            # Add Private Vulnerability and Public Vul Info only
+            message += f"{self.public_vul} known security vulnerability "
+            message += f"and {self.pvt_vul} security advisory with {self.severity[0]} severity. "
+            message += f'Recommendation: use version {self.nocve_version}.'
+            return message
+
+        if self.public_vul:
+            # Add Public Vulnerability Info only
+            message += f"{self.public_vul} known security vulnerability with {self.severity[0]} severity. "
+            message += f'Recommendation: use version {self.nocve_version}.'
+            return message
+
         if self.pvt_vul:
-            # Add Private Vulnerability Info
-            message += f" and {self.pvt_vul} security advisory"
-        message += f" with {self.severity[0]} severity. "                      # Add Severity Info
-        message += f'Recommendation: use version {self.nocve_version}.'     # add recommendation
+            # Add Private Vulnerability Info only
+            message += f"{self.pvt_vul} security advisory"
+            message += f" with {self.severity[0]} severity. "
+        message += f"{self.severity[0]} severity."
```

Comment:
Is this line required  "message += f"{self.severity[0]} severity."" ? (One which is out side of if condition)

Reply:
It shows msg like : 
`"message": "lodash - 4.17.4 has 1 security advisory with medium severity. medium severity.",`

Reply:
Thanks @deepak1725

Reply:
Thank you.! Fixed